jobs/metron_agent/templates/metron_agent_ctl.erb

```diff
@@ -23,8 +23,12 @@ case $1 in
     /var/vcap/packages/metron_agent/syslog_daemon_config/setup_syslog_forwarder.sh /var/vcap/jobs/metron_agent/config
     <% end %>
 
+    <% if p('metron_agent.log_to_syslog') %>
+    tee_output_to_sys_log "$LOG_DIR" metron_agent stdout stderr
```

Comment:
Again, we already `tee_output_to_sys_log "$LOG_DIR"` on line 13 above. Will putting this here provide extra information?